container_registry.go

```diff
@@ -102,6 +112,56 @@ func (s *ContainerRegistryService) ListRegistryRepositories(pid interface{}, opt
 	return repos, resp, err
 }
 
+// GetSingleRegistryRepository gets a detail of single registry repository
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/container_registry.html#get-details-of-a-single-repository
+func (s *ContainerRegistryService) GetSingleRegistryRepository(rid interface{}, opt *GetSingleRegistryRepositoryOptions, options ...RequestOptionFunc) ([]*RegistryRepository, *Response, error) {
+	project, err := parseID(rid)
+	if err != nil {
+		return nil, nil, err
+	}
+	u := fmt.Sprintf("registry/repositories/%s", pathEscape(project))
+
+	req, err := s.client.NewRequest(http.MethodGet, u, opt, options)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	var repos []*RegistryRepository
+	resp, err := s.client.Do(req, &repos)
+	if err != nil {
+		return nil, resp, err
+	}
+
+	return repos, resp, err
+}
+
+// ListGroupRegistryRepositories gets a list of registry repositories in a group.
+//
+// GitLab API docs:
+// https://docs.gitlab.com/ee/api/container_registry.html#within-a-group
+func (s *ContainerRegistryService) ListGroupRegistryRepositories(gid interface{}, opt *ListRegistryRepositoriesOptions, options ...RequestOptionFunc) ([]*RegistryRepository, *Response, error) {
```

Comment:
Please place this method below the other list call. And probably we need to rename that one from `ListRegistryRepositories to `ListProjectRegistryRepositories`. What do you think?